packages/expo-analytics-amplitude/ios/EXAmplitude/EXAmplitude.m

```diff
@@ -80,4 +81,61 @@ - (Amplitude *)amplitudeInstance
   resolve(nil);
 }
 
+UM_EXPORT_METHOD_AS(setTrackingOptions,
```

Comment:
Is it possible that `initializeApiKey:` triggers an event to Amplitude? What my question suggests is: should we add a `trackingOptions` argument to `initialize` method?

Reply:
My 2 arguments why I personally prefer a separate `setTrackingOptions`:
- it's a one-to-one mapping with Amplitude's SDK
- the developer might disable some tracking on init (like IP address, lat/lng), but after the user clicks on "I accept the terms", enable them. So we anyways need a separate `setTrackingOptions` function for this use case.